types/office-js-preview/index.d.ts

```diff
@@ -12370,6 +12370,89 @@ declare namespace Office {
          *                             of type `Office.AsyncResult`. Any errors encountered will be provided in the `asyncResult.error` property.
          */
         prependAsync(data: string, callback?: (asyncResult: Office.AsyncResult<void>) => void): void;
+        /**
+         * Prepends HTML or plain text to the beginning of a message or appointment body when the mail item is sent.
+         * 
+         * To use `prependOnSendAsync`, you must specify a supplementary permission in the manifest. Details vary with the type of manifest. For guidance,
+         * see {@link https://learn.microsoft.com/office/dev/add-ins/outlook/understanding-outlook-add-in-permissions | Understanding Outlook add-in permissions}.
+         * 
+         * @remarks
+         * [Api set: Mailbox preview]
+         * 
+         * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/understanding-outlook-add-in-permissions | Minimum permission level}**: **read/write item**
+         *
+         * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/outlook-add-ins-overview#extension-points | Applicable Outlook mode}**: Compose
+         * 
+         * **Recommended**: Call `getTypeAsync` then pass its returned value to the `options.coercionType` parameter.
+         * 
+         * **Important**: When implementing `prependOnSendAsync`, keep the following in mind.
+         * 
+         * - In a {@link https://learn.microsoft.com/office/dev/add-ins/outlook/smart-alerts-onmessagesend-walkthrough | Smart Alerts add-in},
+         * the prepend-on-send feature runs first.
+         * 
+         * - A new line is added after the prepended content.
+         * 
+         * - If multiple active add-ins call `prependOnSendAsync`, the order of the inserted content depends on the order in which the add-in runs.
+         * The content of the last run add-in appears above previously prepended content.
+         * 
+         * - If the add-in attempts to insert HTML into a plain text body, the content will not be prepended. Conversely, plain text will be inserted into an HTML body.
+         * 
+         * **Errors**:
+         * 
+         * - `DataExceedsMaximumSize`: The `data` parameter exceeds 5,000 characters.
+         * 
+         * - `InvalidFormatError`: The `options.coercionType` parameter is set to `Office.CoercionType.Html`, but the item body is in plain text format.
+         * 
+         * @param data - The string to be prepended to the beginning of the message or appointment body. The string is limited to 5,000 characters.
+         * @param options - An object literal that contains one or more of the following properties:-
```

Comment:
Errant hyphen?
```suggestion
         * @param options - An object literal that contains one or more of the following properties:
```

Reply:
We've had to use this format to list parameter properties. I'll follow up on this with Elizabeth when she returns from holiday.